src/packages/formitem/formitem.taro.tsx

```
@@ -130,30 +133,27 @@
+        if (originTrigger) {
+          originTrigger(...args)
         }
-      })
-    }
+        if (this.props.rules && this.props.rules.length) {
+          dispatch({
+            name: this.props.name,
+          })
+        }
+      }
+    })
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)